docs/developers-guide/e2e-tests.md

```diff
@@ -181,11 +181,9 @@ Cypress._.times(N, () => {
 
 ### Embedding SDK tests
 
-Tests located in `e2e/test/scenarios/embedding-sdk` are used to run automated checks for the Embedding SDK.
+Tests located in `e2e/test-component/scenarios/embedding-sdk/` are used to run automated checks for the Embedding SDK.
 
```

Comment:
I kept this part shorter, it was mostly repeating what we say on the docs for the sdk